site/book/01-getting-started/02-quickstart.md

```diff
@@ -86,14 +99,39 @@ You can preview the changes that will be made to the cluster:
 $ kpt live preview
 ```
 
-Finally, apply the resources to the cluster:
+Apply the resources to the cluster:
 
 ```shell
 $ kpt live apply --reconcile-timeout=15m
 ```
 
 This waits for the resources to be reconciled on the cluster by monitoring their status.
 
+## Fetch updates
+
+On Day n, there might be updates to the remote `nginx` package which you want to rebase your local package against.
+A typical `git rebase` might lead to merge conflicts in this scenario. `kpt pkg update` is semantic-aware
+and intelligently merges local changes with upstream updates.
+
+First, commit your local changes to `git` before update
+
+```shell
+git init; git add .; git commit -am "my-nginx"
+```
+
+```shell
+$ kpt pkg update @v0.2
+```
+
+You can observe that the changes you have made are in-tact along with the new changes from
+upstream(image version updated in `Deployment` resource) are fetched on to local.
```

Comment:
please re-read and audit the new paragraphs. They have grammar issues and missing some words. e.g. "local what?".  Be consistent with language and terminology in the rest of the doc.

Reply:
Updated.